FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -10,13 +10,14 @@ import SwiftUI
 
 struct Dashboard: View {
     @EnvironmentObject var store: Store
+    @EnvironmentObject var router: Router
```

Comment:
wondering if this particular router belongs directly on this view only, rather than as an environmental object. Since popping to root will only ever go back to the dashboard it seems like it could be restricted to here. Just a thought.

Reply:
Are you suggesting this in conjunction with moving router actions into middleware? Intermediate screens currently need to know about the router as well so they can pop a single item off the path to go back.

Reply:
Not necessarily tied to router actions in the middleware. Just thinking that this router could be an @StateObject declared and initialized here if the router is only used in the Dashboard and subviews. Arguably, the same could be said for the store since everything in it applies to the dashboard and subviews, so it's probably fine to keep both declared and initialized one level up in the app. You can disregard this thread ^